### PR TITLE
[JENKINS-35112] Re-implement based on JENKINS-35160 impl in core

### PR DESCRIPTION
**Pull request by stephenc**
See [JENKINS-35112](https://issues.jenkins-ci.org/browse/JENKINS-35112) also https://github.com/jenkinsci/jenkins/pull/2789

@reviewbybees

### CODE REVIEW

File: src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java [hunk 1]
```diff
@@ -38,24 +38,11 @@
...
-import hudson.model.ItemGroupMixIn;
-import hudson.model.Items;
-import hudson.model.Job;
-import hudson.model.ModifiableViewGroup;
-import hudson.model.Run;
-import hudson.model.TaskListener;
-import hudson.model.TopLevelItem;
-import hudson.model.View;
-import hudson.model.ViewGroupMixIn;
+import hudson.model.*;
```
Comment (KostyaSha) [inline src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java]: you didn't follow your own code style rules? ;)

Comment (stephenc) [inline src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java]: Arse ide upgrade screwed me over

Comment (KostyaSha) [inline src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java]: forgot import settings from previous version? :)

Comment (stephenc) [inline src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java]: No more likely they added another knob I need to turn off

**Pull request merged**